api/kyverno/v1/policy_types.go

```diff
@@ -291,6 +291,7 @@ var ConditionOperators = map[string]ConditionOperator{
 // MatchResources is used to specify resource and admission review request data for
 // which a policy rule is applicable.
 type MatchResources struct {
+	RequestTypes []string `json:"requestTypes,omitempty" yaml:"requestTypes,omitempty"`
```

Comment:
It will be good  if we can add some description about the field as well

Reply:
We can also generate the OpenAPIV3 validation as well , using [kubebuider markers ](https://book.kubebuilder.io/reference/markers.html)

Reply:
Currently, I'm having an error with `make kyverno-crd` cmd [discussed here](https://kubernetes.slack.com/archives/C032MM2CH7X/p1644946451610149). I'll update it as soon as this error gets resolved.

Reply:
Can you address this comment?

Reply:
After adding comments for `RequestTypes` I need to run `make kyverno-crd` to update the crds and there's issue with this cmd, we're discussing it on slack, as mentioned earlier, the issue is yet to be resolved.